### ISSUE TITLE
segmentation fault (core dumped) when train.py --hypes ...

### ISSUE THREAD

**Issue by zhihongzeng** [Russell91/TensorBox#147]
Today, I git clone tensorbox, and build the program successfully, but failed to run train.py. 
I found two closed tickets of the similar issue, but the previous tickets does not solve my problem. So I create this new one. Any suggestions would be appreciated. 
 
My working env is:
ubuntu 16.04 LTS, tensorflow version 0.11.head, and no gpu

Here is the output:
python train.py --hypes hypes/overfeat_rezoom.json --gpu 0 --logdir output
WARNING:tensorflow:From train.py:463 in train.: initialize_all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2017-03-02.
Instructions for updating:
Use tf.global_variables_initializer instead.
WARNING:tensorflow:From train.py:472 in train.: all_variables (from tensorflow.python.ops.variables) is deprecated and will be removed after 2016-03-02.
Instructions for updating:
Please use tf.global_variables instead.
Segmentation fault (core dumped)

**Comment by Russell91** [comment]
Does anyone else know what particular tensorflow feature is causing this issue? Always hard to keep up to date with their latest changes.

**Comment by youansheng** [comment]
I have encountered the same fault. You can download the previous version of tensorbox. I have tested the version submitted 20 days ago.

**Comment by zhihongzeng** [comment]
Here is what I did:

git log --oneline
4584ea9 Merge pull request #125 from rakeen/patch-1
cc1a1ab Merge pull request #145 from mrry/patch-1

Then, 
git checkout cc1a1ab
tensorbox/utils$ make && cd ..
python train.py --hypes hypes/overfeat_rezoom.json --gpu 0 --logdir output

The training started ....
Thank you so much.

**Comment by Russell91** [comment]
@zhihongzeng are you guys saying this is an issue with the current version?

**Comment by zhihongzeng** [comment]
Yes. The current version 4584ea9 has this problem while the previous version cc1a1ab does not.
